### DISCUSSION TITLE
[Outdated] How to use this repo

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#4] [Announcements]
**Note: This is now outdated.**  
See the README for instructions on how to use this repo!

**Comment by rebelchris** [comment]
Sidenote, the README link doesn't work here @FredKSchott

**Reply by natemoo-re** [reply]
Thanks! Fixed it. ✅

**Reply by FredKSchott** [reply]
thanks @rebelchris !
